doc/developers/contributing.rst

```diff
@@ -621,12 +621,19 @@ In following example, k is deprecated and renamed to n_clusters::
 
     def example_function(n_clusters=8, k=None):
         if k is not None:
-            warnings.warn("'k' was renamed to n_clusters and will "
-                          "be removed in 0.15.",
+            warnings.warn("'k' was renamed to n_clusters in version 0.13 and "
+                          "will be removed in 0.15.",
                           DeprecationWarning)
             n_clusters = k
 
 
```

Comment:
Ok, will do. I just kept the newlines that were in the original. Removing the first newline would produce a long line, so I'll remove the second:

    warnings.warn("'k' was renamed to n_clusters in version 0.13 and "
                  "will be removed in 0.15.", DeprecationWarning)